Fix broken links to `cssnano` documentation

e1himself opened the pull request:
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** <!-- if bug or feature is checked, this should be too -->
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

https://cssnano.co/docs/optimisations/ is 404 now.
The new location is: https://cssnano.co/docs/what-are-optimisations/

### Breaking Changes

Nope.

### Additional Info

Documentation update.

alexander-akait reviewed (dismissed):
Looks good, can you run prettier on docs?

e1himself commented:
> Looks good, can you run prettier on docs?

Done.

Fixed the prettier complaint. And also changed the commit message to match the rules.

Thanks!
